nvm.psm1

```diff
@@ -341,10 +355,7 @@ function Get-NodeVersions {
     else {
         $nvmPath = Get-NodeInstallLocation
 
-        if (!(Test-Path -Path $nvmPath)) {
-            throw "No Node.js versions have been installed"
-        }
-        else {
+        if (Test-Path -Path $nvmPath) {
```

Comment:
I changed this because the standard behaviour for cmdlets that return a list is to return an empty list if there are no items, instead of throwing a terminating error.

Reply:
Looks like this needs a test to be updated to support the new execution path